packages/grid/src/GridUtils.test.ts

```diff
@@ -226,9 +246,159 @@ describe('start/end range adjustment in one dimension', () => {
       [52, 52],
     ]);
   });
+  it('handles moves within the range', () => {
+    testRange(5, 10, GridUtils.moveItem(9, 6), [
+      [5, 5],
+      [9, 9],
+      [6, 8],
+      [10, 10],
```

Comment:
See this implies the order is maintained, unlike the test case above. If we don't care, we should sort/consolidate. If we do care, then the order should be specific in the first case above as well.